app/models/vm_or_template.rb

```diff
@@ -1544,6 +1542,8 @@ def changed_vm_value?(options)
                    :to => :hardware, :allow_nil => true, :uses => {:hardware => :disks}, :type => :integer
 
   virtual_delegate :provisioned_storage, :to => :hardware, :allow_nil => true, :default => 0, :type => :integer
+  virtual_delegate :num_disks, :to => :hardware, :allow_nil => true, :default => 0, :type => :integer, :uses => {:hardware => :disks}
+  virtual_delegate :num_hard_disks, :to => :hardware, :allow_nil => true, :default => 0, :type => :integer, :uses => {:hardware => :hard_disks}
```

Comment:
wish we didn't need `:uses` here, but delegation to a virtual total doesn't fully specify the uses.

Since we tend to just use these in select, the includes use case is not really a concern

Reply:
This sounds like a future feature.

Reply:
@Fryguy this is a bug introduced by our delayed `load_schema!` changes that @jrafanie and a number of us introduced when we fixed the race condition.